lib/task/digitalobject/digitalObjectRegenDerivativesTask.class.php

```diff
@@ -213,7 +213,16 @@ public static function regenerateDerivatives(&$digitalObject, $options = array()
         break;
 
       default:
-        $usageId = $digitalObject->usageId; // MASTER_ID or EXTERNAL_URI_ID
+        $usageId = $digitalObject->usageId; // MASTER_ID or EXTERNAL_URI_ID or EXTERNAL_FILE_ID
+    }
+
+    // If master isn't stored in AtoM, attempt to cache external resource before
+    // deleting existing derivatives
+    //
+    // An unavailable resource will result in an exception
```

Comment:
It looks better to me without the empty comment line.

Reply:
Fixed.